packages/frontend/src/components/molecules/Header.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import React, { useState, useEffect } from 'react';
+import { FaBars } from 'react-icons/fa';
+import NavBar from './NavBar';
+import { Button } from '../atoms/button';
+import Link from 'next/link';
+function Header() {
+  const [openBar, setOpenBar] = useState(false);
+  const [isMobileScreen, setIsMobileScreen] = useState(false);
+
+  let closeBar = () => setOpenBar(false);
+
+  useEffect(() => {
+    const handleResize = () => {
```

Comment:
tailwind has pre-defined classes to help you design the screen responsiveness
check https://v1.tailwindcss.com/course/making-the-navbar-responsive